src/pages/SubmissionConfig.test.js

```diff
@@ -424,7 +429,7 @@ describe('SubmissionConfig inputs definition', () => {
 
     const cells = within(rows[1]).queryAllByRole('cell')
     expect(cells.length).toBe(5)
-    within(cells[1]).getByText('target_workflow_1.foo.foo_rating')
+    within(cells[1]).getByText('foo_rating_workflow_var')
```

Comment:
- Should we also check that column 0 is `foo`?
- Can we also quickly test a second input without a task (eg `input_name: 'target_workflow_1.bar_rating'`?

Reply:
good ideas. I'll add these checks